Kattis/Juxtaposition.py

```diff
@@ -0,0 +1,9 @@
+import sys
+
+n, m, t = input().split()
+n, m, t = [int(n), int(m), int(t)]
```

Comment:
A nicer way to write this is to combine the two lines into one:
```python
n, m, t = [int(x) for x in input().split()]
```

This is called "list comprehension". You can test it out locally to see if it works.
Reference:
https://www.w3schools.com/python/python_lists_comprehension.asp

Reply:
I pushed both lines of codes into one and deleted the import sys